.github/workflows/ci.yml

```diff
@@ -26,6 +26,7 @@ env:
   PERCY_TOKEN_PIE_DOCS: ${{ secrets.PERCY_TOKEN_PIE_DOCS }}
   PERCY_TOKEN_PIE_BUTTON: ${{ secrets.PERCY_TOKEN_PIE_BUTTON }}
   PERCY_TOKEN_PIE_ICON_BUTTON: ${{ secrets.PERCY_TOKEN_PIE_ICON_BUTTON }}
+  PERCY_TOKEN_PIE_MODAL: ${{ secrets.PERCY_TOKEN_PIE_MODAL }}
```

Comment:
out of curiosity, do we have to create a Percy token for each component?

Reply:
yes that's the current way we need to do it - I'm going to actually raise a separate PR to add this token now :)